src/reducers/Reducers.js

```diff
@@ -0,0 +1,28 @@
+import { handleActions } from 'redux-actions';
+import { requestRestaurants, receiveRestaurants } from "../actions/Actions";
+
+const restaurantsReducer = handleActions(
+{
+        [requestRestaurants]: (state) => (
+        {
+            ...state,
+            isFetching: true,
+        }),
+        [receiveRestaurants]: (state, action) => (
+        {
+            ...state,
+            isFetching: false,
+            error: action.error? action.payload : undefined,
+            items: action.error? [] : action.payload,
+            lastUpdate: Date.now()
```

Comment:
Reducers must be `predictable` which mean you can't use `Date.now()` inside reducers (it will make state different every time it is called)